### PR TITLE
Add CRIS reminders with UAC email templates

### PR DESCRIPTION
**Pull request by hbrace**
# Has the DDL schema changed?
*Pull requests which include schema changes should be labelled with the `schema change` label for visibility*

*The [common entity model pom.xml](ssdc-rm-common-entity-model/pom.xml) file `project.version` must be bumped
appropriately if there are any common entity code changes*

* [x] This PR includes DDL schema changes and has been labelled correctly, bumping to new schema
  version: 
* [ ] The POM has been updated with an appropriate version bump if required

# Motivation and Context

New reminder templates for CRIS to contain UACs

# What has changed

Templates added via patching and to groundzero/dev build
# How to test?

Check it all looks ok
make dev-build
# Links

https://trello.com/c/K8pbtVGz/929-add-uac-to-reminders-5
# Screenshots (if appropriate):

### CODE REVIEW

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
The ground zero template seeding needs to be updated:
https://github.com/ONSdigital/ssdc-rm-ddl/blob/main/groundzero_ddl/packcode_templates/email_templates.sql

**Review by KieranWardle — APPROVED** [review]

File: groundzero_ddl/packcode_templates/email_templates.sql [hunk 1]
```diff
@@ -4,5 +4,7 @@ INSERT INTO casev3.email_template (pack_code, description, notify_template_id, m
...
 ('RE_CY_CRIS', 'CRIS Reminder - Bilingual Welsh', '42af756c-7239-4486-b67d-ce2c6c4a7f9b', null ,'["PORTAL_ID","COLLEX_CLOSE_DATE","FIRST_NAME","__sensitive__.LAST_NAME"]'),
 ('UR_EN_CRIS', 'CRIS UAC Replacement - English', '3543bcf5-6986-4d8d-80ab-79fba4d3f050', null ,'["__uac__","PORTAL_ID","FIRST_NAME","__sensitive__.LAST_NAME"]'),
-('UR_CY_CRIS', 'CRIS UAC Replacement - Bilingual Welsh', 'cfe91e5d-19a9-4325-8e4f-0bcbd1dcacff', null ,'["__uac__","PORTAL_ID","FIRST_NAME","__sensitive__.LAST_NAME"]')
+('UR_CY_CRIS', 'CRIS UAC Replacement - Bilingual Welsh', 'cfe91e5d-19a9-4325-8e4f-0bcbd1dcacff', null ,'["__uac__","PORTAL_ID","FIRST_NAME","__sensitive__.LAST_NAME"]'),
+('RE_UAC_EN_CRIS', 'CRIS Reminder with UAC - English', 'd0224400-4894-4bdd-b0bc-839972641327', null, '["__uac__","PORTAL_ID","COLLEX_CLOSE_DATE","FIRST_NAME","__sensitive__.LAST_NAME"]'),
+('RE_UAC_CY_CRIS', 'CRIS Reminder with UAC - bilingual', 'bc35ebcd-63ca-4a90-aa4c-eea3e85ed506', null, '["__uac__","PORTAL_ID","COLLEX_CLOSE_DATE","FIRST_NAME","__sensitive__.LAST_NAME"]')
```
Comment (AdamHawtin) [inline groundzero_ddl/packcode_templates/email_templates.sql:9]:
```suggestion
('RE_UAC_CY_CRIS', 'CRIS Reminder with UAC - Bilingual', 'bc35ebcd-63ca-4a90-aa4c-eea3e85ed506', null, '["__uac__","PORTAL_ID","COLLEX_CLOSE_DATE","FIRST_NAME","__sensitive__.LAST_NAME"]')
```

File: patches/1100_add_reminder_with_uacs_templates.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+-- ****************************************************************************
+-- Number: 1100
+-- Purpose: Inserts reminder with UAC email templates
+-- Author: Hugh Brace
+-- ****************************************************************************
+
+-- Email Templates
+INSERT INTO casev3.email_template (pack_code, description, notify_template_id, metadata, template) VALUES
+('RE_UAC_EN_CRIS', 'CRIS Reminder with UAC - English', 'd0224400-4894-4bdd-b0bc-839972641327', null, '["__uac__","PORTAL_ID","COLLEX_CLOSE_DATE","FIRST_NAME","__sensitive__.LAST_NAME"]'),
+('RE_UAC_CY_CRIS', 'CRIS Reminder with UAC - bilingual', 'bc35ebcd-63ca-4a90-aa4c-eea3e85ed506', null, '["__uac__","PORTAL_ID","COLLEX_CLOSE_DATE","FIRST_NAME","__sensitive__.LAST_NAME"]')
```
Comment (AdamHawtin) [inline patches/1100_add_reminder_with_uacs_templates.sql:12]:
```suggestion
('RE_UAC_CY_CRIS', 'CRIS Reminder with UAC - Bilingual', 'bc35ebcd-63ca-4a90-aa4c-eea3e85ed506', null, '["__uac__","PORTAL_ID","COLLEX_CLOSE_DATE","FIRST_NAME","__sensitive__.LAST_NAME"]')
```

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
LGTM

**Pull request merged**